images/container-lifecycle.drawio

```diff
@@ -1 +1 @@
uY+jZo17cbfvVUfeZV6SCNSPlP/swkQgzt6jS5fWEv9IyStLWIP8dBkOmqzrzX9feYCtKZqVVlfNWjtEHL0JpAY2hBZ4ZWV6CtQX07Ei0dEotiv9DaRRzjOHw9/prlfWG6pQffoUVQgE7TFWhjWIv19w9Ny260z412Pbs4b1KFptUZNIcbY4C37nwrz573sK0dW99mzecPTrXz7fROvPOtIsmtkUHe4ThaZNAuT4/JMo4IDCpdiZc1Nn/BjjdYs4mV3qYH6lmJHO3Z/+43sut+g0N3nVB7a+XZL9LU0peCafZqu9fsmEUHNZBjGqhGTu951sgRB9NCCgOMyjx+k6Puyj066jyDqjebliZJNi2NgzQ7C97sdsH5bxQuIkjrtbJzfN8wj53F982Wl+ZQgGl4B4zvVU1wn/h+90zzA5z7xfjjOd88elDvgrrVOna6aa+vJL8dHGUJWqExG57BeHQwmkYdjAXyjodGW4PGw5edT7IUvEu2uFdS+Oa/ATDdom+hEOa6g8ZJu12P/beZHfrgv721CP0GlTmfIsg4ds+VuRO2m6N66mEONXbzoJU5u322++29OI4xwxw/KcyKEYU7F/SzNz86KhvHn4EmGR7qMOl1hckdTpqffflP5Mtr8HJHr3bj+pjzUE5863n8N3DiCUVim85e/JTtZSMTH9qd+XA+LH/onQO5/LU8uP4f</diagram><diagram id="XoTICt8-muSg9dlJG_p3" name="Container stop">7Ztbb6M4FIB/TaTdh0QGwu2xSZrZ1XRW1VSjTh9dcAgzBLPGtEl//dpgAhinudJE2+Shwgfb+PKd4+Njt2eMF8svBCbzb9hHUU8H/rJnTHq67lgO+8sFq0KgWYZbSAIS+kJWCR7CNySEQEiz0EdpIyPFOKJh0hR6OI6RRxsySAh+bWab4aj51QQGqCV48GDUlj6GPp2Lfpmgkv+FwmBeflkD4s0ClpmFIJ1DH7/WRMZtzxgTjGnxtFiOUcQHrxyX7wD7y+Ar8gwXPf7d/2qPzMd+Udl0nyLrLhAU0xNXbRV1v8AoEwMmOktX5Qginw2oSGJC5zjAMYxuK+mI4Cz2Ef8OYKkqzx3GCRNqTPgLUboSdMCMYiaa00Uk3s5wTKdwEUacsuw5i2kmpA84Ix4vNKeUEaObxg37wzrJ//AM6SDAOIgQTMJ04OFF/sJL86zTWVEne5RqLRqimSz9gggNGS83URjETPiMKWXVGKOUQkJvOIJMGuMYlbJpGEWirzvOjJhBPpI1KsU8fUF4gShhzQQERZCGL010odCAYJ1vXfQeh+y7OhDaqrtmUUQoa18DJbxlJWk+nqJcnZZtVbmO1Bw2EgGirZrYQ61LlSincR8y7XdBFPNxNHloGdKftecnXtXAFKnJUtScJ1ZlImZd+1lP1ErxZFUsT5XlzkW5viPMGzVhP8rTsjMb84kVpCDonYy6c2q9aVC6N5JOy1j+g2k44y24+fbQ47pmRaxDo2fCngL+hGPeUUQWYcxayFMKqO/gM1t1G1xCMQceG3JE3pscgtLwDT7n9fFZTLhG5l03Rz1zomBBzWHEG3GP0zBvpuLLd1IGylVsZzaEOvPK0FK18osuNBbXxiSLUn0wMIA9LMoeaTDLGS1L4NksRd2YM3f7QlsZM07A6zyk6CGBuSq9MudMWjKZ3o5xhEle1pjmv1ypCf6Nam+0/Le3FrcmauOElK5Ua6V4rfwto3Q05jVfqztNLVe9xmgXqsnh534kFCbK+jfjDtxI6ME6zX1eSGGf5++nxGtkLsw0N9LMRG8w0Ll5XhvntaLVP1DNfSksvpbrKqscaGayrJcobMoD1zxmcmKfYDYMuQZRGMaIlJ0UjSoLHEdakyixdkj4TW4no4nTIWS2K0FmOAOzhZlugTZmQ70ryrSNlKUJjMvRT3DK5jSfs/Xs1N+3ZodtNJLc2Z7N7tn2ZFzYRz7UexqFW9cc6qDX9nl5Wmk8ijzMlQGuYZtA/JR+cmH+1XB8oKPTFXCuZNUcx1IBN1QBB7oCTv9fm7V5xrvA9tjx57RpziXYNGMbYgfO7riaz6Y/LHvMf8CFxzFAEaLoz/b7XXlgk0D3nHQh2up7L0LfzwMfKsqaW9JzRja64nYd26vCDCWjNW41W8Gt0ZlpHCp3Vp8xblWFJwbucNgIUWwJUOSpe0RCNicc/IsMcRlN9mzr4AiXMQDAAevfUKoYSBV3HPDSzW2W91IX7o2mnS3ii4Sb8avNPrfN1lyribdpW+c22aqzhz1dDV3FY0ohzQp1ACSL4zAODgdQDq/xRUBsk+pxugjN6HvUKaJybYiVsH8CNu2WP6Er/AmggFM+3jgdnPYHwMmDAgmb2iucFwynHN60h2dns30OIbHprXexFX7GLP+1icyPLjrYY5+TBmkrb9u6axpMHhDoh6gKfr3HdldEDR05YG6Yin2/piuY0rpbjdsHFN9RkEWQNIMxV49NOk3rCpNWxFtxrKIBuxuH7enp5d4fvS2DH7Pv+LeJf62mP/qqQ5XrnnvPuyLSBnlXck6+kdY0fXCy2yKKyjq8L6KEc/NZzOnilvVDnFbAkr+8hiuP9uAUFwQO1RBRi2G1QbcUtrSrza8S16vpbF4uuhyzKLv7pnRHZHebKNe0Y0CRLSNwVcsmdmd7NFlca9m9Zc0C7KFow0kN9EccLBUG+hp2vBjbq0tRR8v9wIMiJYbD4zHcGtcpzi6vcZ1LJtMAkhG0NEOxv+oorqNEU3UAdFQA5sArgqcf7FaE13Kd1mCvLxLWB9vparDbhw/XcMfu4Y4OEJGjYmt36Wzxjq1HALvctjrQq5mgwoGps6i6Xs7F/CohQITkCex5GSG7Lz5dX5vqghRpO2fZbVJ0VfT0FBfzlKS4rWG97uYuYTfHlg9572/Jd/53vi9SBrSqE3SpphPt5xSNLj+1uW3biuy7p2PJ6j8bi+zV/4cat/8B</diagram></mxfile>
```

Comment:
Couple of comments for the image:
1. It would be better if we could use monospace (Ubuntu monospace?) font for the commands in parantheses to differentiate between regular text and commands in the container start and container stop diagrams.
2. In the Container stop diagram, the last step of the workflow before "Container termination complete", it reads better if we change the text in gray area from "Delete container or an error occurred" to "Container deletion or an error occurred".